### ISSUE TITLE
Support for multiple languages?

### ISSUE THREAD

**Issue by zebradots** [redorescue/redorescue#36]
There are many priorities on the horizon that lay the groundwork for future features to be added. If all of these are in place in the not too distant future, it may be possible to pursue supporting multiple languages, both in the OS and the Redo application itself.

Redo has many users worldwide, and while they have done well reading the English options to make use of the application, there is no doubt that it would be nice if all the strings were available in their native languages. However, adding support for different languages represents a significant investment of time and energy.

If this is something that you feel would be of great benefit, please weigh in below.

**Comment by agnivo007** [comment]
I guess initially, it can be started with the top-10 used languages worldwide, then diversify later...

**Comment by zebradots** [comment]
Yes, this is exactly what we were leaning toward. However, the list might be a bit shorter when it is initially supported (primarily focused on the countries with the most downloads from SourceForge), and will probably support only left-to-right languages at first, in order to keep the focus on adding the basic functionality of i18n itself before getting inundated with a large number of translations to various languages!

**Comment by PizzaProgram** [comment]
Many THANKS for developing this great piece of program!
  
I could help translating to **hungarian** (HUN). But that's not in the "Top10"...
My users I offer to download this ISO to > usually do not speak english well enough to understand everything.

The very best translation system I've seen so far is here:  [Telegram Translation](https://translations.telegram.org/hu/android)
 _(I have no idea how the did that > is it possible to create such a webpage for that, etc... but those "picture-helping system" helps extremely lot.)_  :)

I must share with you guys my experience as developer myself: 
 It was a GREAT mistake when I created my apps "not easily translateable". It backfired later.
So my advice would be:  Think in "finals", do not make "half-jobs".
